### PR TITLE
UT for victims package

### PR DESCRIPTION
**Pull request by camilocot**
Add unit tests for victims package - #41

### CODE REVIEW

File: victims/victims_test.go [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+	podList, e := v.RunningPods(client)
+
+	if e != nil {
+		t.Errorf("Unexpected error %s while getting running pod", e)
+	}
+
+	if len(podList) != 1 {
+		t.Errorf("Expected 1 item in podList, got %d", len(podList))
+	}
+}
```
Comment (asobti) [inline victims/victims_test.go]: Can we also assert that the one pod returned is `app1`

File: victims/victims_test.go [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+	if podList := getPodList(client); len(podList.Items) != 0 {
+		t.Errorf("Expected 0 items in podList, got %d", len(podList.Items))
+	}
+}
+
+func TestDeleteRandomPods(t *testing.T) {
+
+	v := newVictimBase()
+	pod1 := newPod("app1", v1.PodRunning)
+	pod2 := newPod("app2", v1.PodPending)
```
Comment (asobti) [inline victims/victims_test.go]: Do you mean for `app2` to be in `Pending` state or is this just copy/paste?

File: victims/victims_test.go [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+	if e := v.DeleteRandomPods(client, 2); e == nil {
+		t.Error("Expected an error if no pods")
+	}
+}
+
+func TestTerminateAllPods(t *testing.T) {
+
+	v := newVictimBase()
+	pod1 := newPod("app1", v1.PodRunning)
+	pod2 := newPod("app2", v1.PodPending)
```
Comment (asobti) [inline victims/victims_test.go]: Can you remind me the behavior of terminate-all-pods? Does it attempt to terminate pending pods too?

File: victims/victims_test.go [hunk 4]
```diff
@@ -0,0 +1,223 @@
+package victims
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/asobti/kube-monkey/config"
+	"github.com/stretchr/testify/assert"
```
Comment (asobti) [inline victims/victims_test.go]: This new dependency will need to be added to `glide.yaml`.

File: victims/victims_test.go (hunk 3 above)
Comment (camilocot) [reply victims/victims_test.go]: Yes, terminates all the pods under the victim, regardless of their status

File: victims/victims_test.go (hunk 2 above)
Comment (camilocot) [reply victims/victims_test.go]: It verifies that not running pods do not try to be deleted

**Comment by asobti** [comment]
Hey. I'm sorry, but you might have misunderstood my last comment. I'm totally ok with using the `assert` package  - in fact I prefer we use that. But we would need to add it to glide.yaml. If you could revert your last commit and add it to the glide file, this PR is perfect.

**Comment by asobti** [comment]
You would just need to add this

```
diff --git a/glide.yaml b/glide.yaml
index 2c12b908..639e8600 100644
--- a/glide.yaml
+++ b/glide.yaml
@@ -19,3 +19,4 @@ import:
   subpackages:
   - kubernetes
   - rest
+- package: github.com/stretchr/testify
```

**Comment by asobti** [comment]
I went ahead and merged this PR. I revert your last commit, and added the dependency to github.com/stretchr/testify`.

Once again, thanks for this.

**Pull request merged**